holmes/config.py

```diff
@@ -170,20 +170,28 @@ def configure_robusta_ai_model(self) -> None:
                 self._load_default_robusta_config()
                 return
 
-            models = fetch_robusta_models(
+            robusta_models = fetch_robusta_models(
                 self.account_id, self.session_token.get_secret_value()
             )
-            if not models:
+            if not robusta_models:
                 self._load_default_robusta_config()
                 return
 
-            for model in models:
+            for model in robusta_models.models:
                 logging.info(f"Loading Robusta AI model: {model}")
                 self._model_list[model] = {
+                    "name": model,
                     "base_url": f"{ROBUSTA_API_ENDPOINT}/llm/{model}",
                     "is_robusta_model": True,
+                    "model": "gpt-4o",  # Robusta AI model is using openai like API.
```

Comment:
I want to put a thumbs up here, this solves the issue when someone sets the model to a non gpt one also like when someone configures the main model as sonnet.
Github needs a like button for code.